tests/modeltests/tst_formatting.cpp

```diff
@@ -46,9 +46,8 @@ private slots:
         QTest::addColumn<QStringList>("ansiStrings");
         QTest::addColumn<QVector<QVector<QTextLayout::FormatRange>>>("formatting");
 
-        QTest::addRow("no ansi sequence")
-            << QStringList {QStringLiteral(" A  B  C  D  E ")}
-            << QVector<QVector<QTextLayout::FormatRange>> {{{0, 15, {}}}}; // only default formatting
+        QTest::addRow("no ansi sequence") << QStringList {QStringLiteral(" A  B  C  D  E ")}
```

Comment:
unrelated change?

Reply:
yes, I wonder why `clang-format` didn't pick this up